src/nvim/redraw.c

```diff
@@ -0,0 +1,43 @@
+// This is an open source non-commercial project. Dear PVS-Studio, please check
```

Comment:
`redraw.c` is a bit generic. Perhaps a better name for this module is `ui_client.c`